src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -175,15 +175,27 @@
   <data name="AmbiguousImplementationException_NullMessage" xml:space="preserve">
     <value>Ambiguous implementation found.</value>
   </data>
+  <data name="AmbiguousImplementationException_Simple" xml:space="preserve">
+    <value>Ambiguous implementation found for: '{0}'.</value>
+  </data>
+  <data name="AmbiguousImplementationException_Detailed" xml:space="preserve">
+    <value>Ambiguous implementation found for: '{0} {1}'.</value>
+  </data>
   <data name="Arg_AccessException" xml:space="preserve">
     <value>Cannot access member.</value>
   </data>
   <data name="Arg_AccessViolationException" xml:space="preserve">
     <value>Attempted to read or write protected memory. This is often an indication that other memory is corrupt.</value>
   </data>
-  <data name="Arg_AmbiguousMatchException" xml:space="preserve">
+  <data name="Arg_AmbiguousMatchException_NullMessage" xml:space="preserve">
```

Comment:
Renamed this one to match the pattern used by the resx messages for `AmbiguousImplementationExeception_NullMessage`

Reply:
NIT: looks better to be named `Arg_AmbiguousMatchException_NoMessage`, feel free to update the AmbiguousImplementationExeception_NullMessage` too
```suggestion
  <data name="Arg_AmbiguousMatchException_NoMessage" xml:space="preserve">
```